lib/fastlane/actions/dotgpg_environment.rb

```diff
@@ -0,0 +1,46 @@
+module Fastlane
+  module Actions
+    module SharedValues
+    end
+
+    class DotgpgEnvironmentAction < Action
+      def self.run(options)
+        require 'dotgpg/environment'
+
+        if options[:dotgpg_file]
+          dotgpg_file = options[:dotgpg_file]
+        end
+
+        raise "Dotgpg file '#{File.expand_path(dotgpg_file)}' not found".red if dotgpg_file && !File.exist?(dotgpg_file)
```

Comment:
The file validation should be handled in the `verify_block`, take a look at the other actions.